internal/proposals/improvement-of-log-messages-usability.md

```diff
@@ -0,0 +1,75 @@
+# Improvement of log messages usability
+
+## Motivation
+
+We want to provide a common, unified solution to logging across Kyma components, which will enable users and maintainers to easily track specific events happening in Kyma's ecosystem. The solution should be independent of the platform (AWS, Azure, GCP) and easy to introduce in existing components
+
+Users, developers, and maintainers want to have an easy way to track events happening in Kyma, problems that occur, and debug faulty components. This will make understanding of Kyma's ecosystem easier and more transparent
+
+## Solution regarding the log messages
+
+Every mission-critical component (or a critical part of the component) must produce log entries in order to make the problem diagnosis process quick and effective
```

Comment:
I would not narrow that down to the mission-critical components only. Let's say that Every Kyma component should do so. 🙂

Reply:
Yeah :D Done